### PR TITLE
Created mock data for Client

### PR DESCRIPTION
**Pull request by abassey2**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

This creates a mock data folder that is imported into the client for standalone development

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/235

### CODE REVIEW

File: client/src/data/configs.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      id: 3,
+    },
+    {
+      name: "Config 4",
+      description: "This is the fourth config",
+      id: 4,
+    },
+  ];
+
+export default configs;
```
Comment (aflinchb) [inline client/src/data/configs.js]: I see a github linter-related warning (I didn't check yet if there's a js-specific linter in the project yet :) ) on this file, loadClients.js, and loadTests.js saying there's no new line at the end of the file. 

What I'm seeing:
image

**Review by aflinchb — APPROVED** [review]
Approved w/ one comment on adding new lines at the end of the files mentioned.

File: client/src/data/configs.js (hunk 1 above)
Comment (abassey2) [reply client/src/data/configs.js]: We do have eslint installed on the project, but I am not sure why it did not flag that. I will go in and change it!

Comment (aflinchb) [reply client/src/data/configs.js]: It's probably not a default setting with eslint :).  This is one that I've mainly seen popping up on github PR's.

**Pull request merged**